server/src/metadata/tenant-initialisation/standard-objects/users/users.metadata.json

```diff
@@ -0,0 +1,66 @@
+{
+  "nameSingular": "userV2",
+  "namePlural": "usersV2",
+  "labelSingular": "User",
+  "labelPlural": "Users",
+  "targetTableName": "user",
+  "description": "A user",
+  "icon": "business",
```

Comment:
Hey @weiko,
What reference are we using to determine the icon name?

Reply:
@gitstart-twenty good question, since we store a string there I kind of delayed this reflexion until we properly discuss this with the Frontend (which will use that value)
cc @charlesBochet @lucasbordeau should we store the name of the component here?

Reply:
Yes the names are listed in `front/src/modules/display/icons/index.ts` and it goes like : `IconBuildingSkyscraper` and `IconArrowDown` in PascalCase.

Reply:
Adding icons...